### DISCUSSION TITLE
台灣銀行業導入AI 有效提升業務能量與客服效率

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#398] [科技新知]
「麥肯錫分析，AI技術每年可為全球銀行業創造達1兆美元的價值，黃耀漢則認為AI技術的成熟將逐漸改變金融市場生態。」
我認為例如文中提到玉山銀行建立AI模型偵測信用卡交易異常阻擋上億元的方式真的很有AI在科技安全應用的意義，也保護了使用者的權益，AI技術在銀行安全的應用值得更多的發展。

資料來源：https://www.google.com/amp/s/www.chinatimes.com/amp/realtimenews/20220613001820-260410

**Comment by Joan0518** [comment]
希望國泰的人臉辨識系統也可以發展有安全且有效率認證的方式，我今天申辦還要打電話過去，視訊與客服一對一確認，把身分證放到鏡頭裡3到5秒與本人合照截圖等等，覺得很不方便。(好像在寫日記:sweat_smile:)

**Comment by AngusGuei** [comment]
我倒覺得AI技術可以往更有個人信息的方向發展，例如虹膜紋路或聲紋辨識
但這些似乎還在發展中？
